packages/web3-modal/src/components/modals/TransactionsModal/styled.tsx

```diff
@@ -137,13 +137,18 @@ export function statusToConfirmationSpecialColors(status: AnyTransactionReceipt[
   }
 }
 
-export function statusToPillProps(tx: AnyTransactionReceipt): PillProps {
+export function statusToPillProps(
+  tx: AnyTransactionReceipt,
+  cssProps: TransactionsModalTheme['card']['statusPill'] | undefined
+): PillProps {
+  const background = cssProps?.background
+  const text = cssProps?.text
   switch (tx.status) {
     case 'success':
     case 'replaced-success':
       return {
-        backgroundColor: '#708c7e',
-        color: 'ghostwhite',
+        backgroundColor: background?.success ? background.success : '#708c7e',
```

Comment:
this is good but I think the better approach (which you probably missed since it's not documented anywhere) is that you can set the defaults inside `baseTheme.ts` and not have to add a fallthrough case here. The reason it's required to do e.g `background?.success` is that the theme type is set as `Optional` for custom overrides. 

I'd like to fix this in the future so that devs dont have to use the optional chaining operator since the theme at runtime is actually all there. 

In fact, if you check `baseTheme.ts` now i'm sure it's complaining that those props aren't filled in

Reply:
added missing theme fields: [a6dbd93](https://github.com/PAST3LLE/monorepo/pull/77/commits/a6dbd93b858c963840ec86e5c7e0705cf8204721)
removed the ternaries: [4627020](https://github.com/PAST3LLE/monorepo/pull/77/commits/4627020a463a616a66c7c27b03e35b1d998b73fa)